db/user.py

```diff
@@ -23,6 +23,22 @@ def create(musicbrainz_id):
         })
         return result.fetchone()["id"]
 
+def update_token(id):
+    """Update a user's token to a new UUID
+
+    Args:
+        id (int) - the row id of the user to update
+    """
+    with db.engine.connect() as connection:
```

Comment:
You're not catching any possible DB exceptions here. As per usual, catch exceptions and then show an appropriate error message and log an error as well.